README.adoc

```diff
@@ -228,6 +172,69 @@ The reply should be as follows:
 ----
 ====
 
+
+
+== Preparing to Build the Application
+
+To package and run the application, we need to provide an external MySQL database rather than using Spring Boot Docker Compose Support.
+For this task, we can reuse the `compose.yaml` provided with a few modifications.
+First, modify the `ports` entry in `compose.yaml` to be `3306:3306`.
+Second, add a `container_name` of `guide-mysql`.
+After these steps, the `compose.yaml` file should be:
+
+[source,yaml]
+----
+services:
+  mysql:
+    container_name: 'guide-mysql'
+    image: 'mysql:latest'
+    environment:
+      - 'MYSQL_DATABASE=mydatabase'
+      - 'MYSQL_PASSWORD=secret'
+      - 'MYSQL_ROOT_PASSWORD=verysecret'
+      - 'MYSQL_USER=myuser'
+    ports:
+      - '3306:3306'
+----
+You can now run `docker-compose up` to start this MySQL container.
```

Comment:
I'll make the change here.
There is only one other guide I've updated with Docker compose support so far, and I've added an [issue in that repository](https://github.com/spring-guides/gs-messaging-rabbitmq/issues/47). Since it seems that `docker-compose` has an alias to `docker compose`, I'll leave this one open as an ideal contribution for someone new to OSS.